Builder/src/KP2/House.java

```diff
@@ -0,0 +1,53 @@
+package KP2;
+
+public class House implements BuildingBuilder{

```

Comment:
[Low] HouseBuilder would be a most suitable name for this class, as you are not modelating the *House* itself, but the way to build it.